docs/coverage-reporter/uploading-coverage-in-advanced-scenarios.md

```diff
@@ -10,8 +10,10 @@ The following sections include instructions on how to use the Codacy Coverage Re
 
 If your test suite is split in different modules or runs in parallel, you must upload multiple coverage reports for the same language either at once or in sequence.
 
+If one or more coverage reports mark a line as covered multiple times, Codacy still counts it as a single covered line when calculating coverage.
```

Comment:
I wonder if we should specify this elsewhere; it's true for multiple reports but also for single reports covering multiple times.

Reply:
Excellent point. I guess the best place to include this clarification on how Codacy calculates coverage based on the coverage reports could be this line:

https://github.com/codacy/docs/blob/master/docs/faq/code-analysis/which-metrics-does-codacy-calculate.md?plain=1#L126

Reply:
OK, I included it there, reworded it, and simplified given the different context:

```
-   If a line is covered multiple times, Codacy counts it as a single covered line when calculating coverage. 
```